tools/aggchainFEPTools/updateAggregationVkey/updateAggregationVkey.ts

```diff
@@ -0,0 +1,185 @@
+import path = require('path');
```

Comment:
Can we use `getProviderAdjustingMultiplierGas` & `getDeployerFromParameters` from `src/utils.ts` ?